tests/test_wait_signal.py

```diff
@@ -913,27 +913,6 @@ def test_empty_when_no_signal(self, qtbot, signaller):
             pass
         assert blocker.all_signals_and_args == []
 
-    def test_empty_when_no_signal_name_available(self, qtbot, signaller):
```

Comment:
I'm a bit confused by the current situation here (also for the test below) - is this really only relevant for PySide2, or did we just never add PySide6 to the test?

From what this looks like:

https://github.com/pytest-dev/pytest-qt/blob/89178edf898b635a21c59342a0771df55170f160/src/pytestqt/wait_signal.py#L104-L123

and from [[PYSIDE-1911] Provide a way to get the name of a Signal(Instance) - Qt Bug Tracker](https://bugreports.qt.io/browse/PYSIDE-1911) only being partially fixed (I think we never implemented the "extract from repr" part?), I believe those tests should still be run on PySide6 actually?

Reply:
Ahh great catch! You are right, they should be running for PySide 6 too. 

Brought the tests back and updated the condition.